File: AlSpark/src/main/scala/com/al/db/JDBCHelper.scala
package com.al.db

import java.sql.{Connection, DriverManager}

import com.al.config.Config

object JDBCHelper {
  
  def getConnection(): Connection = {
    Class.forName(Config.driver_class)
    return DriverManager.getConnection(Config.db_url, Config.username, Config.password)
	}
  
  def main(args: Array[String]): Unit = {
    JDBCHelper.getConnection()
  }
}

File: AlHadoop/src/main/java/com/al/db/JDBCHelper.java
package com.al.db;

import com.al.config.Config;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class JDBCHelper {
	
	public static Connection getConnection(){
		Connection conn = null;
		try {
			Class.forName(Config.driver_class);
			String url = Config.db_url;
			String username = Config.username;
			String password = Config.password;
			url += "&user=" + username + "&password=" + password;
			System.out.println(url);
			conn = DriverManager.getConnection(url);
		} catch (ClassNotFoundException e) {
			e.printStackTrace();
		} catch (SQLException e) {
			e.printStackTrace();
		}
		return conn;
	}
	
	public static void main(String[] args) {
		JDBCHelper.getConnection();
	}
}
